charts/dragonfly-operator/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Not really sure what this file is for :thinking:

Reply:
https://helm.sh/docs/topics/chart_tests/